### PR TITLE
Bug 1212742 - Setting default height on the ssl-container r=mhenretty

### PR DESCRIPTION
**Pull request by albertopq**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=b48c15c8900e414a3e96424f71bae8e7d8f03122

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=3a2258924bfcc4b004566db87c2b556653f10044

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=9c66153f78bbe6f608fea1c7d06ae42a9617ed70

File: apps/system/js/app_window.js [hunk 1]
```diff
@@ -1196,7 +1196,8 @@
         case 'application-name':
           // Apps have a compulsory name field in their manifest
           // which takes precedence.
-          if (this.manifestURL || this.webManifestURL) {
+          var emptyString = !(detail.content.trim());
```
Comment (mikehenrty) [inline apps/system/js/app_window.js]: Are we sure detail.content is always defined for the metachanged event? I'm just thinking, what if someone had:
<meta name="application-name">
Do we care? Do I care? I don't know the answer.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=bc42f2f9afef5e22f0a32549b3379922324ed534

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=f349be3d0705cf539f58eaae71ad29b6fa2f7fdf

**Pull request merged**